pkg/didcomm/protocol/presentproof/service_test.go

```diff
@@ -278,6 +279,205 @@ func TestService_HandleInbound(t *testing.T) {
 			t.Error("timeout")
 		}
 	})
+
+	t.Run("Receive Propose Presentation (continue)", func(t *testing.T) {
+		var done = make(chan struct{})
+
+		messenger.EXPECT().ReplyTo(gomock.Any(), gomock.Any()).
```

Comment:
all this setup is verbose and distracts from the test case. Any way we can reduce all these lines of setup code in the test case?